airbyte-cdk/python/airbyte_cdk/sources/declarative/transformations/remove_fields.py

```diff
@@ -55,7 +60,11 @@ def transform(
         for pointer in self.field_pointers:
             # the dpath library by default doesn't delete fields from arrays
             try:
-                dpath.util.delete(record, pointer)
+                dpath.util.delete(
```

Comment:
can you add some unit tests showing how this is meant to be used?
- no condition
- with a condition that evaluates to false
- with a condition that evaluates to true
- with a condition that depends on the config

Reply:
sure, will do

Reply:
added